app/pages/GrossWeightPage.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import generated.GoodsMeasureType03
+import pages.sections.OtherQuestionsSection
+import play.api.libs.json.JsPath
+
+case object GrossWeightPage extends DiscrepancyQuestionPage[BigDecimal, Option[GoodsMeasureType03], BigDecimal] {
```

Comment:
Remove and use `pages.grossMass.GrossMassPage` instead. More than happy for you to move that file if you think it's silly to have a `grossMass` folder

Reply:
@jackheslop96 made changes and moved folders, also renamed page to gross weight page